Dockerfile

```diff
@@ -72,4 +72,10 @@ EXPOSE 4000
 
 HEALTHCHECK CMD curl --fail http://localhost:4000 || exit 1  
 
-CMD ["bin/membrane_videoroom_demo", "start"]
+COPY --chown=nobody:nobody docker-entrypoint.sh ./docker-entrypoint.sh
+
+RUN chmod +x docker-entrypoint.sh
+
+ENTRYPOINT ["./docker-entrypoint.sh"]
+
+CMD ["bin/membrane_videoroom_demo", "start"]
```

Comment:
Is it a kind of a pattern to specify an entrypoint with a script which executes some desired additional command, and pass the actual entrypoint command as entrypoint arguments with `CMD` command, so that it can be executed as the last instruction of the entrypoint script? Personally I would simply append the `bin/membrane_videoroom_demo start` command at the end of the `docker-entrypoint.sh` and remove the `CMD` from the `Dockerfile`, but perhaps it is not the advised way.